DataFormats/Scouting/interface/Run3ScoutingElectron.h

```diff
@@ -0,0 +1,131 @@
+#ifndef DataFormats_Run3ScoutingElectron_h
+#define DataFormats_Run3ScoutingElectron_h
+
+#include <vector>
+
+// Class for holding electron information, for use in data scouting
+// IMPORTANT: the content of this class should be changed only in backwards compatible ways!
+class Run3ScoutingElectron {
+public:
+  //constructor with values for all data fields
+  Run3ScoutingElectron(float pt,
+                       float eta,
+                       float phi,
+                       float m,
+                       float d0,
+                       float dz,
+                       float dEtaIn,
+                       float dPhiIn,
+                       float sigmaIetaIeta,
+                       float hOverE,
+                       float ooEMOop,
+                       int missingHits,
+                       int charge,
+                       float ecalIso,
+                       float hcalIso,
+                       float trackIso,
+                       float r9,
+                       float sMin,
+                       float sMaj,
+                       unsigned int seedId,
+                       std::vector<float> energyMatrix,
+                       std::vector<float> timingMatrix)
+      : pt_(pt),
+        eta_(eta),
+        phi_(phi),
+        m_(m),
+        d0_(d0),
+        dz_(dz),
+        dEtaIn_(dEtaIn),
+        dPhiIn_(dPhiIn),
+        sigmaIetaIeta_(sigmaIetaIeta),
+        hOverE_(hOverE),
+        ooEMOop_(ooEMOop),
+        missingHits_(missingHits),
+        charge_(charge),
+        ecalIso_(ecalIso),
+        hcalIso_(hcalIso),
+        trackIso_(trackIso),
+        r9_(r9),
+        sMin_(sMin),
+        sMaj_(sMaj),
+        seedId_(seedId),
+        energyMatrix_(std::move(energyMatrix)),
+        timingMatrix_(std::move(timingMatrix)) {}
+  //default constructor
+  Run3ScoutingElectron()
+      : pt_(0),
+        eta_(0),
+        phi_(0),
+        m_(0),
+        d0_(0),
+        dz_(0),
+        dEtaIn_(0),
+        dPhiIn_(0),
+        sigmaIetaIeta_(0),
+        hOverE_(0),
+        ooEMOop_(0),
+        missingHits_(0),
+        charge_(0),
+        ecalIso_(0),
+        hcalIso_(0),
+        trackIso_(0),
+        r9_(0),
+        sMin_(0),
+        sMaj_(0),
+        seedId_(0),
+        energyMatrix_(0),
+        timingMatrix_(0) {}
```

Comment:
Implicit calls to default constructors are sufficient
```suggestion
        seedId_(0) {}
```